spec/spec_helper.rb

```diff
@@ -17,7 +17,7 @@
 require 'rack/test'
 require 'rspec'
 require "json"
-
+require 'logstash/runner'
```

Comment:
It's weird that all our specs require random bits of LS, and potentially dangerous for our testing as well.